README.md

```diff
@@ -217,14 +217,16 @@ kubectl create -f scope-probe-ds.yaml
 
 ## <a name="developing"></a>Developing
 
-The build is in two stages. `make deps` installs some tools we use later in
-the build. `make` builds the UI build container, builds the UI in said
-container, builds the backend build container, builds the app and probe in a
-said container, and finally pushes the lot into a Docker image called
-**weaveworks/scope**.
+Building Scope from source depends on the latest version of
```

Comment:
Probably not but scope doesn't really target anything other than the latest docker, so may as well have them upgrade it.